acceptance/tests/apply_plan_ssh.rb

```diff
@@ -36,30 +36,6 @@
     on(ssh_nodes, "rm -rf #{filepath}")
   end
 
-  step "execute `bolt plan run noop=true` via SSH with json output" do
-    result = bolt_command_on(bolt, bolt_command + ' noop=true', flags)
-    assert_equal(0, result.exit_code,
-                 "Bolt did not exit with exit code 0")
-
-    begin
-      json = JSON.parse(result.stdout)
-    rescue JSON::ParserError
-      assert_equal("Output should be JSON", result.string,
-                   "Output should be JSON")
-    end
-
-    ssh_nodes.each do |node|
-      # Verify that node succeeded
-      host = node.hostname
-      result = json.select { |n| n['node'] == host }
-      assert_equal('success', result[0]['status'],
-                   "The task did not succeed on #{host}")
-
-      # Verify that files were not created on the target
-      on(node, "cat #{filepath}/hello.txt", acceptable_exit_codes: [1])
-    end
-  end
-
```

Comment:
This test seemed redundant since we already test that a plan with `apply` works and we test that `noop` works at the integration level.